BizHawk.Client.Common/lua/EmuLuaLibrary.Events.cs

```diff
@@ -51,89 +51,77 @@ public void CallExitEvent(Lua thread)
 		public void CallSaveStateEvent(string name)
 		{
 			var lfs = _luaFunctions.Where(l => l.Event == "OnSavestateSave").ToList();
-			if (lfs.Any())
+			try

```

Comment:
What if there's nothing attached?

Reply:
If `Where` doesn't match anything, `lfs` will be an empty list. The `foreach` has nothing to iterate over so it can't throw, and the `catch` block won't be entered.